src/screens/CreatePost/index.js

```diff
@@ -309,20 +311,13 @@ const CreatePost = () => {
 
   const uploadPhotoImage = async (pathImg) => {
     try {
-      const compressionResult = await Image.compress(pathImg, {
-        compressionMethod: 'auto'
-      });
-
+      const compressionResult = await ImageCompressionUtils.compress(pathImg);
       const newArr = {
         id: mediaStorage.length,
         data: compressionResult
       };
 
-      const asset = new FormData();
-      asset.append('photo', composeImageMeta(compressionResult));
-
-      const responseUpload = await uploadPhoto(asset);
-
+      const responseUpload = await ImageUtils.uploadImage(pathImg);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code seems to be missing the part where the compressed image data is added to a FormData object before being uploaded. This could potentially cause issues if the `ImageUtils.uploadImage` function expects its argument to be a FormData object containing the image data under a specific key (like 'photo' in the old code). If this is the case, you should create a FormData object and append the compressed image data to it before calling `ImageUtils.uploadImage`.

```diff
  const uploadPhotoImage = async (pathImg) => {
    try {
      const compressionResult = await ImageCompressionUtils.compress(pathImg);
      const newArr = {
        id: mediaStorage.length,
        data: compressionResult
      };
  
+     const asset = new FormData();
+     asset.append('photo', composeImageMeta(compressionResult));
+ 
-     const responseUpload = await ImageUtils.uploadImage(pathImg);
+     const responseUpload = await ImageUtils.uploadImage(asset);
      setMediaStorage((val) => [...val, newArr]);
      setDataImage((val) => [...val, responseUpload.data.url]);
      sheetMediaRef.current.close();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
We already used FormData and call composeImageMeta inside ImageUtils.uploadImage. So no need use this